### ISSUE TITLE
Error ValidationError: file validation failed: url: Path `url` is required.

### ISSUE THREAD

**Issue by rafiandria23** [Lith/strapi-provider-upload-google-cloud-storage#57]
**Describe the bug**
I tried to uploads images in Strapi Admin Panel, using this upload provider, but when I clicked the upload button, it showed `Internal Server Error`

**Expected behavior**
Successfully uploads images to GCS Bucket.

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04 LTS
 - Browser: Google Chrome

**Comment by Lith** [comment]
Hi ! 

Could you provide more information, like Strapi version, node, npm ?

Have you tried to upload file using local storage ?

Strapi console log output more information about the error, if you could paste it this will help me to help you.

Thanks :)

**Comment by kalanyuz** [comment]
Got the same error trying to do the same with strapi-3.0.5:node12 with 3.0.1 version of this module.
Local upload works normally

```
error ValidationError: file validation failed: url: Path `url` is required.
strapi    |     at new ValidationError (/srv/app/node_modules/mongoose/lib/error/validation.js:31:11)
strapi    |     at model.Document.invalidate (/srv/app/node_modules/mongoose/lib/document.js:2428:32)
strapi    |     at p.doValidate.skipSchemaValidators (/srv/app/node_modules/mongoose/lib/document.js:2277:17)
strapi    |     at /srv/app/node_modules/mongoose/lib/schematype.js:1064:9
strapi    |     at processTicksAndRejections (internal/process/task_queues.js:79:11)
```

**Comment by Lith** [comment]
Hi @kalanyuz 👋 

Yes, we can add strapi stderr log on catch part of the code (it’s a good improvement).

Does your problem resolved with a correct bucket name ?

Thanks,

**Comment by kalanyuz** [comment]
@Lith yep. That solves the problem for me.
Other things could cause hard to debug issues too (ex. having a space at the end of `baseUrl` or `basePath` in config, where this would not raise any issues, path gets saved to the database, and the uploaded files become unreachable)
#54 might solves a lot of these

**Comment by Lith** [comment]
Fixed #60
